src/pretix/plugins/badges/templates.py

```diff
@@ -0,0 +1,322 @@
+#
+# This file is part of pretix (Community Edition).
+#
+# Copyright (C) 2014-2020 Raphael Michel and contributors
+# Copyright (C) 2020-2021 rami.io GmbH and contributors
+#
+# This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation in version 3 of the License.
+#
+# ADDITIONAL TERMS APPLY: Pursuant to Section 7 of the GNU Affero General Public License, additional terms are
+# applicable granting you additional permissions and placing additional restrictions on your usage of this software.
+# Please refer to the pretix LICENSE file to obtain the full terms applicable to this work. If you did not receive
+# this file, see <https://pretix.eu/about/en/license>.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License along with this program.  If not, see
+# <https://www.gnu.org/licenses/>.
+from django.utils.translation import gettext_lazy as _
+from reportlab.lib import pagesizes
+from reportlab.lib.units import mm
+
```

Comment:
Would it make sense to add helper-dicts for attendee-name and attendee-company, which we could reuse in creating the layouts?

```suggestion
layout_part_attendee = {
    "type": "textarea",
    "page": 1,
    "locale": "",
    "left": "13.09",
    "bottom": "49.73",
    "fontsize": "23.6",
    "lineheight": 1,
    "color": [0, 0, 0, 1],
    "fontfamily": "Open Sans",
    "bold": True,
    "italic": False,
    "width": "121.83",
    "downward": False,
    "content": "attendee_name",
    "text": "John Doe",
    "text_i18n": {},
    "rotation": 0,
    "align": "center",
}
```

and later reuse it with e.g.

`{**layout_part_attendee, **{"left": "5.00", "bottom": "10"}}`

Or with an even nicer dev-„UI“ using a function that does the merging for you and is called like:

```
get_layout_part("attendee", {
    "left": "100.00",
    "bottom": "20.73"
})
```